text/0000-project-groups.md

```diff
@@ -0,0 +1,391 @@
+  present.
+- If there is consensus to become active again, go to step 3.
+
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[internals]: https://internals.rust-lang.org
+
+# Reference-level explanation
+
+## Common Aspects of Working Groups and Project Groups
+
+Before going into the differences between these groups it's important to start
+with what it is shared between them.
+
+- A charter that defines the scope and intent of the group.
+
+- A GitHub repository hosted under the `rust-lang` organization
+  (or separate organisation if necessary) containing the charter and
+  instructions for how community members can monitor or participate in
+  the group.
+
+- Groups have at least one shepherd who acts as the leader of the group and is
+  typically responsible for writing the initial charter, handling administrative
+  and communication tasks, as well as delegating responsibilities to other
+  members in the group.
+
+- Groups should have a liaison member associated with an official Rust team.
+
+  - This liaison may also be but is not required to be a shepherd.
+
+- Groups and their members are represented on the official
+  [rust-lang.org] website.
+
+- Membership requirements for both groups is decided by the shepherd and should
+  be stated in the charter.
+
+  - Initial membership should try to represent people who have already been
+    participating regularly and productively in the respective area.
+
+- Neither group has _"formal decision making power"_: meaning that they are not
+  able to accept RFCs on `rust-lang/rfcs`. Similarly, neither group has
+  representation on the Core team.
+
+  - Groups are of course encouraged to create RFCs as well
+    as advocate their concerns and desired changes to the Rust teams
+    and community.
+
+- Both groups are entitled to spaces in any or all of Rust's officially managed
+  discussion forums.
+
+  - As the time of this writing this includes [Zulip] and [Discord].
+  - Groups are also free to create spaces on platforms where the Rust project
+    does not have a official presence. Moderation for these spaces should still
+    follow Rust's [Code of Conduct].
+
+[rust-lang.org]: https://rust-lang.org
+[zulip]: https://rust-lang.zulipchat.com
+[discord]: https://discord.gg/rust-lang
+[code of conduct]: https://www.rust-lang.org/policies/code-of-conduct
+
+### Initial Setup
+
+Once a group has been approved, a pull request with the initial set of members
+should be made to [`rust-lang/team`]. Please refer to team's documentation for
+how to create a group.
+
+It is recommended for the working group to create a `wg-{name}`/`project-{name}`
+repository under the `rust-lang` organisation using the [project group
+template], and making any relevant changes and personalisation.
+
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[project group template]: https://github.com/rust-lang/project-group-template
+
+## Working Groups
```

Comment:
I think this RFC should focus on project groups only and we can have another for evolving the current domain WGs. I think the two are fairly orthogonal and having a smaller RFC should make discussion quicker and more focussed.

Reply:
Yeah, in re-reading the RFC I tend to agree, it'll also make it easier to read.